
geometry_msgs/Pose2D  pose
float32 confidence
bool    isHandUp